server/models/BookingRequest.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const bookingRequestSchema = new mongoose.Schema({
+  user_id: {
```

Comment:
Please `camelCase` this variable

Reply:
resolved by b23bbcc204aa31c6333eb8863c307917cf05b870